packages/common/src/types/chainUpdate.ts

```diff
@@ -87,6 +96,26 @@ export interface BakerStakeThresholdUpdate {
     update: BakerStakeThreshold;
 }
 
+export interface TimeoutParametersUpdate {
+    updateType: UpdateType.TimeoutParameters;
+    update: TimeoutParameters;
+}
+
+export interface MinBlockTimeUpdate {
+    updateType: UpdateType.MinBlockTime;
+    update: Duration;
+}
+
+export interface BlockEnergyLimitUpdate {
+    updateType: UpdateType.BlockEnergyLimit;
+    update: bigint;
+}
+
+export interface FinalizationCommitteeParametersUpdate {
+    updateType: UpdateType.FinalizationCommitteeParameters;
+    update: FinalizationCommitteeParameters;
+}
```

Comment:
Please, please add documentation for all public stuff.

Reply:
Yes, I agree.. I will add doc comments for all the stuff pertaining to the changes implemented here, and will add an issue to document the rest, as it will probably take a substantial amount of time.

Reply:
Added this issue for reference: https://github.com/Concordium/concordium-node-sdk-js/issues/210

Reply:
> Yes, I agree.. I will add doc comments for all the stuff pertaining to the changes implemented here, and will add an issue to document the rest, as it will probably take a substantial amount of time.

We can probably "cheat" and reuse the already quite well documented GRPC-types for most of them right? It will still take some time, but at least we don't have to write it from scratch for a good amount of them, right?